### PR TITLE
add kubectl run deploy pod into specific namespace

### PR DESCRIPTION
**Pull request by cloudmelon**
>^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> Remember to delete this note before submitting your pull request.
>
> For pull requests on 1.15 Features: set Milestone to 1.15 and Base Branch to dev-1.15
> 
> For pull requests on Chinese localization, set Base Branch to release-1.14
>
> For pull requests on Korean Localization: set Base Branch to dev-1.14-ko.\<latest team milestone>
>
> If you need Help on editing and submitting pull requests, visit:
> https://kubernetes.io/docs/contribute/start/#improve-existing-content.
>
> If you need Help on choosing which branch to use, visit:
> https://kubernetes.io/docs/contribute/start#choose-which-git-branch-to-use.
>^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
>

### CODE REVIEW

**Comment by cloudmelon** [comment]
@bradamant3 Please review my changes thanks :)

Best wishes from the UK
Melony

File: content/en/docs/reference/kubectl/cheatsheet.md [hunk 1]
```diff
@@ -290,6 +290,10 @@ kubectl logs -f my-pod                              # stream pod logs (stdout)
 kubectl logs -f my-pod -c my-container              # stream pod container logs (stdout, multi-container case)
 kubectl logs -f -l name=myLabel --all-containers    # stream all pods logs with label name=myLabel (stdout)
 kubectl run -i --tty busybox --image=busybox -- sh  # Run pod as interactive shell
+kubectl run nginx --image=nginx --restart=Never -n  # Run pod nginx in a specific namespace
```
Comment (xiangpengzhao) [inline content/en/docs/reference/kubectl/cheatsheet.md]: The below `mynamespace` should follows `-n` here.

Comment (cloudmelon) [reply content/en/docs/reference/kubectl/cheatsheet.md]: @xiangpengzhao  Please review my changes thanks :)

Comment (xiangpengzhao) [reply content/en/docs/reference/kubectl/cheatsheet.md]: The change LGTM.
/cc @erictune @krousey 
for another review.

**Pull request merged**